install.sh

```diff
@@ -1,8 +1,9 @@
 #!/bin/bash
 set -e # everything must succeed.
-if [ ! -d venv ]; then
-    virtualenv --python=`which python2` venv
+rm -rf venv/
+if [ ! -e venv3/bin/activate ]; then
+    rm -rf venv3/
+    python -m venv venv3
 fi
-source venv/bin/activate
-
-pip install -r requirements.txt
+source venv3/bin/activate
```

Comment:
it seems the script from https://github.com/elifesciences/lax-formula/blob/ca880a4232eb5755244f458e125769faf1fae4ee/salt/lax/cron.sls#L26 in turn uses `install.sh` so the change to `venv3` should be completely encapsulated here without propagating into the formula